sycl/plugins/hip/pi_hip.cpp

```diff
@@ -909,16 +897,25 @@ pi_result hip_piextDeviceSelectBinary(pi_device device,
 
 pi_result hip_piextGetDeviceFunctionPointer(pi_device device,
                                             pi_program program,
-                                            const char *function_name,
```

Comment:
I am not sure I understand the need for this change? Not that it is critical in any way, but the next argument uses "function_..." in the name so it does make it a little less consistent.

Reply:
I see! Then I guess the consistency comment applies to both backends. Again it is in no way critical but if you're in there changing something else, feel free to rename them. Feel free to resolve this discussion if not. 

I wonder if there is a good way to unify all the common implementation between the HIP and CUDA backend. Could make maintenance easier. Though that is way outside the scope of this PR!

Reply:
Yes, ill tidy up the naming while I am at it. Thanks